cmd/loadgenerator/main.go

```diff
@@ -69,14 +69,14 @@ func findRootKeys(algodDir string) []*crypto.SignatureSecrets {
 		var handle db.Accessor
 		handle, err := db.MakeErasableAccessor(path)
 		if err != nil {
-			return nil //nolint:nilerr // don't care, move on
+			return nil
```

Comment:
Why is the nilerr linter willing to allow this now?

Reply:
```
cmd/loadgenerator/main.go:72:15: directive `//nolint:nilerr // don't care, move on` is unused for linter "nilerr" (nolintlint)
			return nil //nolint:nilerr // don't care, move on
			           ^
cmd/loadgenerator/main.go:79:15: directive `//nolint:nilerr // don't care, move on` is unused for linter "nilerr" (nolintlint)
			return nil //nolint:nilerr // don't care, move on
```

Reply:
It started warning me for having it, warning went away when I removed. Given what this code does, I wasn't too concerned.

Reply:
Yes, the code is certainly correct. But I don't see how `nilerr` knows that. It's doing exactly the thing `nilerr` thinks is bad, because it usually is.
@cce knows linters well. Any ideas?

Reply:
Confirmed that removing //nolint: nilerr from alt locations such as data/transactions/logic/eval.go:4266 results in linter failures.